WS2012R2/lisa/xml/STOR_VHDX.xml

```diff
@@ -178,6 +179,22 @@
            <timeout>800</timeout>
        </test>
 
+       <test>
+           <testName>VHDx_DiskHotPlugUnplug_Multi_SCSI</testName>
+           <preTest>
+               <file>setupscripts\AddVhdxHardDisk.ps1</file>
```

Comment:
Hi @OvidiuRusu , 
This script is used to test hot add and remove Disk  64 times, so we add these two script in pretest.